cli/tools/bundle/esbuild.rs

```diff
@@ -44,6 +44,10 @@ pub async fn ensure_esbuild(
   tarball_cache: &Arc<TarballCache<CliNpmCacheHttpClient, CliSys>>,
   npm_cache: &CliNpmCache,
 ) -> Result<PathBuf, AnyError> {
+  if let Ok(esbuild_path) = std::env::var("ESBUILD_PATH") {
+    return Ok(PathBuf::from(esbuild_path));
+  }
```

Comment:
I thought we already had a way to overwrite this? Maybe it should be done in another PR and documented?